src/course-unit/course-xblock/xblock-content/iframe-wrapper/iframe-wrapper.js

```diff
@@ -0,0 +1,368 @@
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/cms/js/require-config.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/bundles/js/factories/context_course.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/bundles/js/sock.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/bundles/js/factories/container.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/debug_toolbar/js/toolbar.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/vendor/url.min.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/vendor/URI.min.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/models/xblock_info.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/views/xblock.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/views/utils/xblock_utils.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/common/js/components/utils/view_utils.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/utils/module.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/views/baseview.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/common/js/components/views/feedback_notification.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/common/js/components/views/feedback_prompt.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/utils/handle_iframe_binding.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/utils/templates.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/common/js/components/views/feedback.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/js/vendor/requirejs/text.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/xblock/resource/split_test/public/js/split_test_author_view.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/bundles/WordCloudBlockDisplay.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/xblock/resource/library_content/public/js/library_content_edit.js"></script>
+      <script type="text/javascript" src="${studioBaseUrl}/static/studio/bundles/HtmlBlockDisplay.js"></script>
+      <script type="text/javascript" src="https://app.getbeamer.com/js/beamer-embed.js"></script>
+      <script type="text/javascript" src="https://studio.edx.org/c4x/edX/DemoX/asset/jquery.loupeAndLightbox.js"></script>
```

Comment:
No wonder it's slow to load each iframe!

Reply:
Surely not all of these are required. e.g. Three of them are just for `drag-and-drop-v2` and definitely not used by any other XBlocks.

Reply:
Right. The frontend would have to make a request to the backend to ask it which JS files are needed for a given XBlock (or XBlock type).

Reply:
I thought that when the backend renders the XBlock, it will include the "common" JS plus whatever JS that specific XBlock has requested (e.g. `virtual-dom` for drag and drop). This is still mostly being rendered by the backend right?